machines/bleShare/scan/scanMachine.ts

```diff
@@ -78,17 +78,21 @@ const model = createModel(
     QrLoginRef: {} as ActorRefFrom<typeof qrLoginMachine>,
     linkCode: '',
     readyForBluetoothStateCheck: false,
+    showFaceCaptureSuccessBanner: false,
   },
   {
     events: {
       SELECT_VC: (vc: VC) => ({vc}),
       SCAN: (params: string) => ({params}),
-      ACCEPT_REQUEST: () => ({}),
+      ACCEPT_REQUEST: (isFaceIDVerificationFlow: boolean) => ({
```

Comment:
Are we having "ACCEPT_REQUEST" event triggers with different param values - true / false?
https://github.com/mosip/inji/pull/1270/files#diff-8644aa2a28639628e4d0ed0920d026bd4470b5b24616264cc144401f39332192R60

If not, Do we need isFaceIDVerificationFlow param here, if we are always passing it up as false?

Reply:
Removed the param and passing the data directly now.